lang/java/reef-common/src/main/java/org/apache/reef/driver/evaluator/EvaluatorRequest.java

```diff
@@ -147,7 +147,7 @@ public String getRuntimeName() {
     private final List<String> rackNames = new ArrayList<>();
     private String runtimeName = "";
 
-    private Builder() {
+    public Builder() {
```

Comment:
We would need a default constructor for this `Builder` which we can access from outside, but I thought this would be the simplest way to enable that. At the moment, I'm unsure if there are other ways to achieve this, but I'm open to any kinds of suggestions.